### DISCUSSION TITLE
Symbolic link to git repo response very slow when theme is on.

### DISCUSSION THREAD

**Discussion by aneesurrehman001** [JanDeDobbeleer/oh-my-posh#2697] [Troubleshoot]
Hi,

I have a symbolic link to a cloned git repository in my root directory on WSL/Ubuntu. When I try "cd" to that symbolic link it takes too much time to move into that git repo when the theme is already applied. But if I try "cd entire-path-to-git-repo" it works fine. When I remove the theme the symbolic link to git repo also works fine. Symbolic link to regular directories also work fine when the theme is on.

Looks like there is some issue with theme and git repo symbolic link.

Thanks!

**Comment by JanDeDobbeleer** [comment]
@aneesurrehman001 you can have a look at that location to see where oh-my-posh spends its time. Navigate to that folder and run `oh-my-posh debug`.

**Reply by aneesurrehman001** [reply]
![Screenshot](https://user-images.githubusercontent.com/55150455/186192448-66442c72-810f-4463-9d64-48ee08d36da7.png)
@JanDeDobbeleer

**Reply by JanDeDobbeleer** [reply]
@aneesurrehman001 yes, so git is the culprit. Which command takes that much time? Please dump the full output (redacted) below.
